Tests/DIOSteps.fcl

```diff
@@ -1,8 +1,8 @@
 #include "Production/JobConfig/primary/DIOtail.fcl"
 source.firstRun: 501 #TODO
-physics.filters.TargetStopResampler.fileNames : [
-"/pnfs/mu2e/tape/phy-sim/sim/mu2e/MuminusStopsCat/MDC2020k/art/39/6a/sim.mu2e.MuminusStopsCat.MDC2020k.001201_00000000.art"]
-# NOTE: to use on the grid do samweb list-file-locations --schema=root --defname="sim.mu2e.MuminusStopsCat.MDC2020k.art" | cut -f1 and place that file name here
+physics.filters.TargetStopResampler.fileNames : ["root://fndca1.fnal.gov:1094/pnfs/fnal.gov/usr/mu2e/tape/phy-sim/sim/mu2e/MuminusStopsCat/Mu2eIIa2/art/40/89/sim.mu2e.MuminusStopsCat.Mu2eIIa2.001201_00000000.art"]
+
+#"/pnfs/mu2e/tape/phy-sim/sim/mu2e/MuminusStopsCat/Mu2eIIa2/art/40/89/sim.mu2e.MuminusStopsCat.Mu2eIIa2.001201_00000000.art"
```

Comment:
Two things.  

The title of this commit is gitignore but the commit actually changes a different file.  Please confirm your intentions.

Second.  Is this code in active developement?  If so it's OK to leave in commented out lines for short time.  If not then please remove the commented out lines.

Reply:
This is an update to the stopped muon file

Reply:
Yes, I see that.  I am asking why does the commit comment say "gitignore" when the commit does not modify the .gitignore file?  We discussed modifying .gitignore  to exclude *.root and *.art files and I was expecting to see that.

Reply:
Thanks for updating .gitignore.